pkg/logger/logger.go

```diff
@@ -568,3 +568,7 @@ func GetLevel() Level {
 func HasDebugLevel() bool {
 	return pkgLogger.cfg.Level == DebugLevel
 }
+
+func Current() *Logger {
+	return pkgLogger
+}
```

Comment:
If the question below is true, this is not required.

Reply:
Perhaps a comment letting the caller know that it's receiving a pointer so direct changes to the returned logger will affect the whole system.